task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -635,3 +681,11 @@ def stop(self) -> None:
 
     def get_event_queue(self) -> "Queue[Event]":
         return self.event_queue
+
+    @property
+    def is_stopping(self):
+        return self.stopping
+
+    @is_stopping.setter
+    def is_stopping(self, val):
+        self.stopping = val
```

Comment:
I was having issues with writing a test for `_reconcile_task_loop` as it was entering an infinite loop while waiting for `self.stopping` to turn true during the test case. In order to avoid this I mock the is_stopping function here in order to simulate only 1 loop and exit out after the second loop. https://github.com/Yelp/task_processing/blob/766c7e058d1cde9401f6bab4d58d4ee582e1ead0/tests/unit/plugins/kubernetes/kubernetes_pod_executor_test.py#L359-L361

Reply:
great idea